packer/plugin.go

```diff
@@ -268,4 +269,26 @@ type PluginDetails struct {
 	PluginPath  string
 }
 
-var PluginsDetailsStorage = map[string]PluginDetails{}
+type PluginsDetailsStorage struct {
```

Comment:
This type doesn't need to be exported as it is used only within this scope. Since our usage exposes a package level var we have all the access we need.